Sources/PerformanceTesting/PerformanceTestCase.swift

```diff
@@ -87,6 +110,20 @@ open class PerformanceTestCase: XCTestCase {
         XCTAssert(results.correlation >= minimumCorrelation)
     }
 
+    public func assertPerformance(
+        _ complexity: Complexity,
+        _ operation: (Int) -> Double
```

Comment:
Could we make the API: `assertPerformance(_:of:)`?

In the case that we are passing around a named function, it could read as: 

```Swift
assertPerformance(.factorial, of: permutations)
```
 whereas if we define the closure in-place, it the preposition is omitted inherently:

```Swift
assertPerformance(.constant) { print("fuck Generals") }
```